tensorflow/lite/micro/test_helpers.h

```diff
@@ -216,7 +216,6 @@ TfLiteTensor CreateTensor(const T* data, TfLiteIntArray* dims,
   result.is_variable = is_variable;
   result.allocation_type = kTfLiteMemNone;
   result.data.data = const_cast<T*>(data);
-  result.quantization = {kTfLiteAffineQuantization, nullptr};
```

Comment:
Oof, thanks for catching this!

Reply:
I had a hard time tracking down where this line came from, and it seems like an error. I couldn't find any cases that assumed this incorrect behavior and tests are passing with this fix.